utilities/aylienApi.js

```diff
@@ -0,0 +1,64 @@
+const AylienNewsApi = require('aylien-news-api');
+const epoch = require('epoch.js');
+
+const formatStories = require('./aylienApiDataFormatter');
+
+const getStories = (peaks, queryString, callback) => {
```

Comment:
Not sure if intentionally deviating from the interface here of `getNews(searchString, peaks, (peakStories) => {` https://github.com/TrendGame/TrendGame/blob/master/utilities/makeTimeline.js#L10